amlb/datasets/openml.py

```diff
@@ -218,16 +230,36 @@ def _get_split_path_for_dataset(ds_path, split='train', fold=0):
 
 
 @profile(logger=log)
-def _save_split_set(path, name, full_dataset=None, indexes=None):
+def _save_split_set(path, name, full_dataset=None, rows=None, cols=None):
     # X_split = X[indexes, :]
     # y_split = y.reshape(-1, 1)[indexes, :]
     log.debug("Saving %s split dataset to %s.", name, path)
+    if rows is None:
+        rows = slice(None)
+    else:
+        rows = np.array(rows)
+    full_attributes = full_dataset['attributes']
+    if cols is None:
+        cols = slice(None)
+        attributes = full_attributes
+    elif isinstance(cols, list):
+        cols = np.array(cols)
+        attributes = [full_attributes[i] for i in cols]
+    else:
```

Comment:
Which scenario is this? `cols` is only ever a `list` (in the way it is called right now), I understand the `None` option since it is still the default (even if it is never left at default). Am I missing something?

Reply:
also not necessary anymore: tried to pass `slice` as params at some point.
will simplify